server/shared/configFetcher.go

```diff
@@ -13,27 +13,27 @@ import (
 	"github.com/woodpecker-ci/woodpecker/server/remote"
 )
 
-type configFetcher struct {
-	remote_ remote.Remote
-	user    *model.User
-	repo    *model.Repo
-	build   *model.Build
+type ConfigFetcher struct {
```

Comment:
why export this struct?

Reply:
Ah forgot to mention this in the summary. Because of this rule: https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#unexported-return